README.md

```diff
@@ -1,18 +1,18 @@
 # OpenDSSDirect.py
-[![Travis Build Status](https://travis-ci.org/NREL/OpenDSSDirect.py.svg?branch=master)](https://travis-ci.org/NREL/OpenDSSDirect.py)
-[![Appveyor Build Status](https://ci.appveyor.com/api/projects/status/github/NREL/OpenDSSDirect.py?branch=master&svg=true)](https://ci.appveyor.com/project/kdheepak/opendssdirect-py)
-[![codecov](https://codecov.io/gh/NREL/OpenDSSDirect.py/branch/master/graph/badge.svg)](https://codecov.io/gh/NREL/OpenDSSDirect.py)
+[![Travis Build Status](https://travis-ci.com/dss-extensions/OpenDSSDirect.py.svg?branch=master)](https://travis-ci.com/dss-extensions/OpenDSSDirect.py)
+[![Appveyor Build Status](https://ci.appveyor.com/api/projects/status/github/dss-extensions/OpenDSSDirect.py?branch=master&svg=true)](https://ci.appveyor.com/project/PMeira/opendssdirect-py)
```

Comment:
@kdheepak I didn't know it would move the project to my instance when I imported it. I can remove it from mine so you readd it, if you prefer.

Reply:
I think Appveyor projects have to be on some user account and cannot be on a github organization name. So I don't mind it being on yours. It can be on yours across all the projects for consistency.